### PR TITLE
[Misc] Support jmh tests

### PR DESCRIPTION
**Pull request by sendaoYan**
Summary: Support jmh tests

Testing: CI pipeline

Reviewers: Accelerator1996, ziyilin

Issue: https://github.com/dragonwell-project/dragonwell11/issues/898

### CODE REVIEW

**Review by ziyilin — COMMENTED** [review]
LGTM

**Review by ziyilin — APPROVED** [review]

File: test/micro/org/openjdk/bench/java/security/CacheBench.java [hunk 1]
```diff
@@ -44,7 +44,7 @@
 @OutputTimeUnit(TimeUnit.NANOSECONDS)
 @Fork(jvmArgsAppend = {"--add-exports", "java.base/sun.security.util=ALL-UNNAMED", "-Xmx1g"})
 public class CacheBench {
-
+/*
```
Comment (Accelerator1996) [inline test/micro/org/openjdk/bench/java/security/CacheBench.java:47]: 既然代码无用，为何选择注释而不是删除

**Review by Accelerator1996 — APPROVED** [review]

File: test/micro/org/openjdk/bench/java/security/CacheBench.java (hunk 1 above)
Comment (sendaoYan) [reply test/micro/org/openjdk/bench/java/security/CacheBench.java:47]: 层风老师建议注释掉，而不是删除

**Comment by sendaoYan** [comment]
Thanks all for the reviews.

**Pull request merged**